api/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from .views import AddressListView, AddressCreateView, AddressRetrieveView
+
+urlpatterns = [
+    path('api/v1/addresses/view', AddressListView.as_view(), name='address-list'),
+    path('api/v1/addresses/create', AddressCreateView.as_view(), name='address-create'),
+    path('api/v1/addresses/view/<int:pk>', AddressRetrieveView.as_view(), name='address-retrieve'),
```

Comment:
`int` or `bigint`? why not a `uuid`?

Reply:
Based on the requirements we can change the int to bigint or uuid. The reason for not using uuid is to easily test the api endpoints as uuid are complex to read and remember.